sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -761,3 +761,79 @@ ALTER TABLE `facility` ADD `uuid` binary(16) DEFAULT NULL;
 #IfNotIndex facility uuid
 CREATE UNIQUE INDEX `uuid` ON `facility` (`uuid`);
 #EndIf
+
+#IfNotRow codes code_text "respiratory syncytial virus monoclonal antibody (motavizumab), intramuscular"
```

Comment:
would remove the quotes above (the function will collect all text there including spaces

Reply:
below this will need to collect the code_type being used for cvx and then use it in the queries below.
Should look something like:
```
SET @codetypeid = (SELECT `ct_id` FROM  `code_types` WHERE `ct_key` = 'CVX');
```